index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <script
+      src="https://kit.fontawesome.com/8ea4cd9e84.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <header id="header">
+      <img
+        id="header-img"
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+        alt="original trombones logo"
+      />
+      <nav id="nav-bar">
+        <div class="nav-link">
+          <a href="#Features">Features</a>
+          <a href="#How It Works">How It Works</a>
+          <a href="#Pricing">Pricing</a>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <div class="title">
+        <h2>Handcrafted, home-made masterpieces</h2>
+      </div>
+      <div class="email">
+        <input
+          type="email"
+          id="email"
+          name="email"
+          placeholder="Enter your email address"
+        />
```

Comment:
Good job on creating well descriptive input  however it would be great  to use a one line code for the inputs and button. Unless required. This keeps the code neat and readable.